articles/machine-learning/how-to-prepare-datasets-for-automl-images.md

```diff
@@ -47,36 +47,12 @@ It helps to create, manage, and monitor data labeling tasks for
 
 If you already have a data labeling project and you want to use that data, you can [export your labeled data as an Azure ML Dataset](how-to-create-image-labeling-projects.md#export-the-labels) and then access the dataset under 'Datasets' tab in Azure ML Studio. This exported dataset can then be passed as an input using `azureml:<tabulardataset_name>:<version>` format. Here is an example on how to pass existing dataset as input for training computer vision models.
 
-# [Azure CLI](#tab/cli)
+### Using pre-labeled training data from local machine
+If you have previously labeled data that you would like to use to train your model, you will first need to upload the images to the default Azure Blob Storage of your Azure ML Workspace and register it as a [data asset](how-to-create-data-assets.md). 
 
-[!INCLUDE [cli v2](../../includes/machine-learning-cli-v2.md)]
+Below scripts uploads the image data on your local machine at path "./data/odFridgeObjects" to datastore in Azure Blob Storage. Thereafter, it creates a new data asset with the name "fridge-items-images-object-detection" in your Azure ML Workspace. 
 
-```yaml
```

Comment:
This seems to be a bad merge. We should not remove the content about using datasets from labeling. Please fix this.

Reply:
I am not sure, if I understood correctly. I have only renamed the title of section to " Using pre-labeled training data from local machine". Original para below it is as-is